website/docs/components/button-set/index.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: ButtonSet
+description: Provides consistent layout and spacing for a set of buttons.
+caption: Group of buttons.
```

Comment:
I'm not sold on this caption being so short, but I'll have to see it along with the rest of the cards once these are merged up. 

```suggestion
caption: A set of buttons.
```